platform/docs/docs/platform/modes/index.md

```diff
@@ -80,6 +80,7 @@ function modeFactory() {
     hangingProtocol: [],
     sopClassHandlers: [],
     hotkeys: [],
+    labelConfig: {}
```

Comment:
I believe we should opt for a customization service for this instead of hardcoding it here.